### PR TITLE
Create badge component with label, labelColor, backgroundColor, borderColor args

### PR DESCRIPTION
**Pull request by nickgraz**
The `labelColor` will match the `borderColor` unless specified explicitly.

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#475** <a href="https://app.graphite.dev/github/pr/terraware/web-components/475?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/web-components/475?utm_source=stack-comment).

**Comment by nickgraz** [comment]
publish rc

**Review by karthikbtf — APPROVED** [review]
Nice 👍

**Review by joshbuchea — APPROVED** [review]
Are all badges currently one size?

Changes are clean and straight forward, looks good 👍

**Comment by nickgraz** [comment]
> Are all badges currently one size?
> 
> Changes are clean and straight forward, looks good 👍

Currently, I think so. Only two places where they exist (that I'm aware of) for now. Can easily modify this later if we need more customization

**Comment by karthikbtf** [comment]
> > Are all badges currently one size?
> > Changes are clean and straight forward, looks good 👍
> 
> Currently, I think so. Only two places where they exist (that I'm aware of) for now. Can easily modify this later if we need more customization

We have badges in some existing places (accessions status, draft badge in planting sites list) - all of them have custom implementations.. will be good to replace them all with this badge. I believe the size should work for those use-cases.

File: src/components/Badge/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+
+export interface BadgeProps {
+  backgroundColor?: string;
+  borderColor?: string;
+  label: string;
```
[the remark this answers is not in the document]
Comment (nickgraz) [reply src/components/Badge/index.tsx:7]: Added that, defaults to `borderColor` but can be defined explicitly

[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/Badge/index.tsx:7]: Awesome, thanks 🙏

**Pull request merged**